source/configure/experimental-configuration-settings.rst

```diff
@@ -660,6 +660,19 @@ See the `Important Upgrade Notes <https://docs.mattermost.com/upgrade/important-
 | This feature's ``config.json`` setting is ``"PatchPluginsReactDOM": false`` with options ``true`` and ``false``. |
 +------------------------------------------------------------------------------------------------------------------+
 
+Disable data refetching on browser refocus
```

Comment:
@M-ZubairAhmed Is this config setting available for both self-hosted and Cloud instances? And is it available in Team Edition, or is Professional or Enterprise needed?

Reply:
@cwarnermm sorry for delay on this.
This setting is available on both self and colour. And its available in all plans starting from "Free"